lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -10,18 +24,16 @@
 
       <div class="wrapper-help">
         <div class="help help-task photo-tips facetips">
-          <h4 class="title"><%- gettext( "Tips on taking a successful photo" ) %></h4>
+          <h4 class="title"><%- gettext( "To take a successful photo, make sure that:" ) %></h4>
 
           <div class="copy">
             <ul class="list-help">
-              <li class="help-item"><%- gettext( "Make sure your face is well-lit" ) %></li>
-              <li class="help-item"><%- gettext( "Be sure your entire face is inside the frame" ) %></li>
-              <li class="help-item">
-                <%=  _.sprintf( gettext( "Once in position, use the camera button %(icon)s to capture your photo" ), { icon: '<span class="example">(<i class="icon fa fa-camera" aria-hidden="true"></i>)</span>' } ) %>
-              </li>
-              <li class="help-item"><%- gettext( "Can we match the photo you took with the one on your ID?" ) %></li>
-              <li class="help-item"><%- gettext( "Use the retake photo button if you are not pleased with your photo" ) %></li>
+              <li class="help-item"><%- gettext( "Your face is well-lit." ) %></li>
+              <li class="help-item"><%- gettext( "Your entire face fits inside the frame." ) %></li>
+              <li class="help-item"><%- gettext( "The photo of your face matches the photo on your ID." ) %></li>
             </ul>
+
+            <p class="copy-extra"><%=  _.sprintf( gettext( "To use the current photo, select the camera button %(icon)s. To take another photo, select the retake button %(icon)s." ), { icon: '<span class="example">(<i class="icon fa fa-camera" aria-hidden="true"></i>)</span>' } ) %></p>
```

Comment:
It doesn't look like there is any accessible text for the camera icons that are being swapped in here.  Try adding:
```<span class="sr">icon</span>``` after the icon font.  Make sure its not inside the ```aria-hidden```